### ISSUE TITLE
support for response 201-204

### ISSUE THREAD

**Issue by wdrost** [lscalese/OpenAPI-Client-Gen#16]
Method ensResponseClass checks on response 200, but response can also be 201 - 204, so should be checked

**Comment by lscalese** [comment]
Yes, You're right.
I check how to integrate this.

**Comment by lscalese** [comment]
@wdrost It's okay!
I add a parsing response for all status codes.  

Example:
```
Class sid1.msg.patchworksAtResponse Extends sid1.msg.ParsedResponse [ ProcedureBlock ]
{

Parameter "202ISLIST" = 0;

Parameter "202MODEL" = "WorksAt";

Parameter "202MODELTYPE" = "sid1.model.Definition.WorksAt";

Property WorksAt As sid1.model.Definition.WorksAt;
```

In addition, If a service has multiple object defined (schema."$ref") in different status code, the generated class response contain a property for each different type.  

Utils:processParsedResponse check the status code and parse the related property.  



It will included in version 1.0.3
